indico/modules/events/editing/client/js/components/ExtensionList.jsx

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Dropdown} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+export default function ExtensionList({value, disabled, onChange, onFocus, onBlur}) {
+  const handleChange = (e, {value: newValue}) => {
+    onChange(_.uniq(newValue.map(x => x.replace(/\W/g, ''))));
```

Comment:
Already mentioned offline, but for the same of completeness: This breaks e.g. `tar.gz` extensions; the only things to change here is stripping surrounding whitespace (if the dropdown component doesn't already do it) and leading dots.